packages/components/nodes/documentloaders/Unstructured/UnstructuredFile.ts

```diff
@@ -118,44 +281,57 @@ class UnstructuredFile_DocumentLoaders implements INode {
     async init(nodeData: INodeData, _: string, options: ICommonObject): Promise<any> {
         const filePath = nodeData.inputs?.filePath as string
         const unstructuredAPIUrl = nodeData.inputs?.unstructuredAPIUrl as string
-        const elementType = nodeData.inputs?.elementType as string
+        const strategy = nodeData.inputs?.strategy as UnstructuredLoaderStrategy
+        const encoding = nodeData.inputs?.encoding as string
+        const coordinates = nodeData.inputs?.coordinates as boolean
+        const xmlKeepTags = nodeData.inputs?.xmlKeepTags as boolean
+        const skipInferTableTypes = nodeData.inputs?.skipInferTableTypes as SkipInferTableTypes[]
```

Comment:
for `multiOptions`, should we do `JSON.parse(skipInferTableTypes)` ?

Reply:
Oh good catch. I made the change and tested.
Before: `"skip_infer_table_types": ["\"[\\\"pdf\\\",\\\"jpg\\\",\\\"pptx\\\"]\""],`
After: `"skip_infer_table_types": ["pdf", "jpg", "pptx"],`